util/keto/ready.go

```diff
@@ -16,7 +17,7 @@ func IsReady() {
 	_, err := client.Do(req)
 
 	if err != nil {
-		util.Log.Error("Cannot connect to Keto Server")
+		loggerx.Error(errors.New("Cannot connect to Keto Server"))
```

Comment:
log err and not "Cannot connect to Keto Server"

Reply:
I think it loggerx should be removed from here because loggerx.Error is getting panic in IsReady() because here there is no req obj that is required for logging error.